frontend/src/lib/uPlotLib/utils/getYAxisScale.ts

```diff
@@ -9,7 +9,7 @@ function findMinMaxValues(data: QueryDataV3[]): [number, number] {
 		entry.series?.forEach((series) => {
 			series.values.forEach((valueObj) => {
 				const value = parseFloat(valueObj.value);
-				if (!value) return;
+				if (Number.isNaN(value)) return;
```

Comment:
@Rajat-Dabade : Let compute min / max only if the value is of type number

Reply:
That's what we are doing here. Series can have NaN along with numbers, so numbers should be compared and NaN's should be skipped.

Reply:
It is possible that the value can be of any type, making sure the value is of type number guarantees that there are no edge case breaks.

Reply:
Sure.